app/redux/reducers/users.js

```diff
@@ -0,0 +1,46 @@
+const SET_SELECTED_USER = 'SET_SELECTED_USER';
+const SET_GROUP_USERS = 'SET_GROUP_USERS'
+
+const defaultState = {
+  selectedUser: {},
```

Comment:
Instead of selectedUser, can we call this 'viewer'?
We cannot select a user in our app.  Instead the person using the app is just the viewer :-D